### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by yl-msft**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    jobs:
+      - job: Build
+        timeoutInMinutes: 30
+        strategy:
+          maxParallel: 8
+          matrix:
+            Debug x86:
+              BuildConfiguration: Debug
+              BuildPlatform: x86
+              BuildPlatformDir: Win32
```
Comment (brycehutchings) [inline azure-pipelines.yml]: BuildPlatformDir isn't needed/used

Comment (brycehutchings) [reply azure-pipelines.yml]: PublishMui and RunVSTest too

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            inputs:
+              solution: Samples.sln
+              vsVersion: "16.0"
+              msbuildArgs: '/fl /flp:LogFile=$(System.DefaultWorkingDirectory)\bin\Logs\MsBuild_$(BuildConfiguration)_$(BuildPlatform).log;Verbosity=diagnostic'
+              platform: $(BuildPlatform)
+              configuration: $(BuildConfiguration)
+              maximumCpuCount: true
+
+          - task: VSBuild@1
+            displayName: "Build BasicXrApp.sln"
```
Comment (brycehutchings) [inline azure-pipelines.yml]: Will this be merged into Samples.sln?

Comment (yl-msft) [reply azure-pipelines.yml]: So far, this BasicXrApp is stand alone, not using XrSceneLib.  Maybe keep it as is so it's the simple MVP sample without extra dependencies.

Comment (brycehutchings) [reply azure-pipelines.yml]: Ah ok sounds good

**Pull request merged**